exam/src/main/java/in/testpress/exam/ui/TestQuestionFragment.java

```diff
@@ -168,6 +170,29 @@ private String getQuestionItemHtml() {
         return htmlContent;
     }
 
+    private String generateHeaderHtml(AttemptQuestion attemptQuestion){
```

Comment:
Header is implicit here, because in a question it is difficult to know which one is header.
So you can move the index back and extract marks alone.